pymanifold/lib.py

```diff
@@ -149,3 +151,87 @@ def _create_market(
             headers=self._auth_headers(),
         )
         return LiteMarket.from_dict(response.json())
+    
+    def kelly_calc(self, slug, subjective_probability, balance, initial_market_probability=0.5):
```

Comment:
I think it might make more sense to have this be in a `utils` package (e.g. `pymanifold.utils.kelly_calc`), and have the function accept a market -- since this function is more a calculation on a market, rather than an action needing to perform an API action

e.g. `kelly_calc(market, subjective_probability, balance, initial_market_probability)`.

What do you think?

Reply:
I'm a maths student and might be unfamiliar with software engineering standards. I'm not sure how to create a `utils ` package (or why that would be advisable). Could you please elaborate?

Your second point is well taken and shouldn't be too hard.

Reply:
I have changed the `kelly_calc` method to accept a market object rather than a slug.